packages/flutter/lib/src/material/icons.dart

```diff
@@ -38,7 +38,7 @@ class IconData {
 ///
 ///  * [Icon]
 ///  * [IconButton]
-///  * <https://design.google.com/icons/>
+///  * [design.google.com/icons](https://design.google.com/icons/)
```

Comment:
How much better does this kind of thing look in the docs? Is there any way we can do it automatically in the dartdoc generator? It makes the source look a lot uglier...